x-pack/plugins/observability_solution/investigate_app/public/pages/details/components/events_timeline/events_timeline.tsx

```diff
@@ -14,20 +14,23 @@ import { getBrushData } from '@kbn/observability-utils-browser/chart/utils';
 import { Group } from '@kbn/observability-alerting-rule-utils';
 import { ALERT_GROUP } from '@kbn/rule-data-utils';
 import { SERVICE_NAME } from '@kbn/observability-shared-plugin/common';
+import { v4 } from 'uuid';
+import { EventSchema } from '@kbn/investigation-shared';
 import { AnnotationEvent } from './annotation_event';
 import { TIME_LINE_THEME } from './timeline_theme';
 import { useFetchEvents } from '../../../../hooks/use_fetch_events';
 import { useInvestigation } from '../../contexts/investigation_context';
 import { useKibana } from '../../../../hooks/use_kibana';
 import { AlertEvent } from './alert_event';
+import { LLMEvent } from './llm_event';
```

Comment:
this should probably be more specific. if you want the root cause analysis events, those types are exported from `service_rca/index.ts` iirc.

Reply:
I have renamed `LLMEvent` to `RootCauseAnalysisEvent`, and event type from `llm` to `rca`. I am mapping RCA significant event [here](https://github.com/dgieselaar/kibana/blob/c0733ebb45d296d7229b811733e14508163af5d5/x-pack/plugins/observability_solution/investigate_app/public/pages/details/components/events_timeline/events_timeline.tsx#L85-L91) to existing one we are using for events timeline.